src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -860,6 +855,14 @@ private TypeSymbol BindVariableType(CSharpSyntaxNode declarationNode, Diagnostic
                     hasErrors = true;
                 }
             }
+            else if (kind == LocalDeclarationKind.ReadOnlyVariable)
```

Comment:
I assume fixed and using variables cannot be declared readonly. Is this correct? Testing this?